### PR TITLE
Add new stories in French

### PR DESCRIPTION
**Pull request by adeclosset**
@octaveauger - can you review?

### CODE REVIEW

**Comment by clippy** [comment]
**Changing existing CSS?**

Make sure you know that your changes to existing CSS aren't having any adverse or unexpected consequences across the app you're working with - this should involve tracking down *any* instance of the class, tag or otherwise that you're changing and checking it out locally to make sure everything looks :ok_hand:.

The way things appear affects how GoCardless is perceived - think more "Tech? Internet? We got this!" and less "Your money is safe with us, probably".

**Adding new CSS?**

If you're adding a new component, make sure your naming of both the filename and classes is consistent and follows our conventions:

Here's what's :+1::

```
/* hero.css */

/* The parent class for your component */
.hero {}

/* Child elements denoted by double undescore */
.hero__image {}

/* Modified versions (which should still depend on their unmodified counterparts)
denoted by double hyphen */
.hero__image--large {}
```

Here's what's :-1::

```
/* page-hero.css */

/* Doesn't match filename */
.hero {}

/* Parent-child relationship suggested by hyphen instead of double underscore */  
.hero-image {}

/* No, just no. */
.hero-image__large {}
```

**Comment by octaveauger** [comment]
@adeclosset this might need a bit of design love, no? The Habitat box on the left feels suboptimal.
image

**Comment by adeclosset** [comment]
@octaveauger - Would be nice to center it temporarily ( @sebastiengocardless is working on a new reference from Mutinerie). 

@waltfy - favour to ask, could you help center habitat reference?

File: app/pages/stories/stories.fr.js [hunk 1]
```diff
@@ -65,6 +67,15 @@ export default class Stories extends React.Component {
         </div>
 
         <hr className='u-margin-An' />
+        <div className='site-container u-padding-Vxxl'>
+          <div className='grid u-padding-Vxl'>
+            <StoryLink imgClass='habitat-logo stories-item__logo--habitat' to='stories_habitat'>
```
Comment (adeclosset) [inline app/pages/stories/stories.fr.js]: @harrison - Hey Phil, this is the issue. It's quite hard to change StoryLink, for exemple to allow centering of this div or to make it larger. Right now, this story link is alone and so is hanging out on the left on the FR website (see screenshot of Octave in the PR). 

@waltfy suggested maybe modifying story-link.js to allow adding css to this element. What do you think is the best way to do this?

**Comment by harrison** [comment]
@adeclosset sorry chatted to @jamesshedden and seems this is more of design discussion, even having it centered just looks a bit weird so maybe this page should be rethought a bit, want to ping @jamesshedden about it?

**Comment by adeclosset** [comment]
@harrison @jamesshedden - I'd like to get this on the website sooner rather than later and no add extra work to the team. 

So I suggest this: we'll have another reference coming up this week and we'll add one from the UK website to have a nice symmetry. Sounds good?

**Comment by jamesshedden** [comment]
@adeclosset A row of 3 would be fine if you're happy to add the new one this week etc before merging this! Otherwise let me know - as it currently stands the design doesn't make a lot of sense with one story in isolation looking like this, so we'd want to figure that out before merging.

**Comment by adeclosset** [comment]
@octaveauger - can you review translation of Foundation of Hearts?

**Comment by octaveauger** [comment]
@adeclosset done

**Comment by adeclosset** [comment]
Hey guys, would like to commit this, getting the third reference is taking longer than expected and this branch is getting old. We'll update as soon as possible.

@Ken-Lo - can you have a quick check and merge?

**Comment by Ken-Lo** [comment]
@adeclosset Awesome work! :+1: Have fixed the conflict, merged in latest changes and tidied up the French stories page so each link takes up half of the screen (we can change this back when you get a third story added)

screen shot 2015-12-07 at 16 25 05

Happy to merge this if everyone else is happy :)

**Comment by Ken-Lo** [comment]
Hi guys,
Just had this checked over by Amaury so going to merge this in now.
@adeclosset When you guys add a third story, you can take out `cssSizeClass='u-size-1of2'` to get it back to the old look with 3 stories per line.

**Pull request merged**